### ISSUE TITLE
LoadingCache.getAll doesn't invoke loadAll

### ISSUE THREAD

**Issue by acjay** [feijoas/mango#13]
From what I can see, [`LoadingCache.getAll`](https://github.com/feijoas/mango/blob/master/src/main/scala/org/feijoas/mango/common/cache/LoadingCache.scala#L125-L155) doesn't seem to actually invoke `CacheLoader.loadAll`, as it states in the docs. Unless I'm missing some clever indirection.

**Comment by mschneiderwng** [comment]
This is correct. The default implementation of [`LoadingCache.getAll`](https://github.com/feijoas/mango/blob/master/src/main/scala/org/feijoas/mango/common/cache/LoadingCache.scala#L125-L155) does not match the documentation and should be fixed.

However, the only implementation is [`LoadingCacheWrapper`](https://github.com/feijoas/mango/blob/master/src/main/scala/org/feijoas/mango/common/cache/LoadingCacheWrapper.scala#L50) which overrides this method calling [guava](https://github.com/google/guava/blob/master/guava/src/com/google/common/cache/LoadingCache.java#L119).
